### PR TITLE
Create codeql-analysis.yml

### PR DESCRIPTION
**Pull request by sangwoo108**
issue #3 : setup codeql

### CODE REVIEW

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
```
Comment (simonhong) [inline .github/workflows/codeql-analysis.yml:2]: I don't know about yml file. Is this file template?

Comment (sangwoo108) [reply .github/workflows/codeql-analysis.yml:2]: Yeah, it's a kind of markup language and used by some frameworks.
If we locate this file, CodeQL scanning for PRs will start to wrok.
image

Comment (sangwoo108) [reply .github/workflows/codeql-analysis.yml:2]: And as for the contents of this file, yes it's a template provided by Github.

Comment (simonhong) [reply .github/workflows/codeql-analysis.yml:2]: Got it!

**Review by simonhong — APPROVED** [review]

**Comment by sangwoo108** [comment]
Thank you for reviewing this!

**Pull request merged**